src/libraries/Errors.sol

```diff
@@ -0,0 +1,89 @@
+/**
+ * Created by Pragma Labs
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+pragma solidity 0.8.19;
```

Comment:
Doesn't having all errors in a single contract leads to all errors being deployed with each contract, also the ones not used (eg. errors of LendingPool in a Tranche?

Reply:
No only internal functions of a library would be added to the contract (in order to use JUMP opcode instead of DELEGATECALL). Otherwise you only have to deploy the library once and it will not be included in the bytecode of the contract deployed. Only the library's address for external reference. That's how I understand it.

Reply:
Yes, but isn't the full library added to the bytecode of each Contract that inherits it?

Reply:
In that case to check: does the deploy-script deploy the external library first and automatically link it to the same external library?